packages/components/src/InputText/InputText.tsx

```diff
@@ -27,7 +38,7 @@ interface MultilineProps extends BaseProps {
   /**
    * Specifies the visible height of a long answer form field.
```

Comment:
Probably should update the comment to include the new option for this prop here.

Reply:
Good call thanks!